test/diff.js

```diff
@@ -35,6 +39,7 @@ void (
   for (const [lhs, rhs] of [
     [true, false],
     [-42, 42],
+    [int(-43), int(43)],
```

Comment:
Let's take this out of the array, and use the `diffsBoxedPrimitives` macro only if `BigInt` is available. We should make sure the title clarifies we're testing a BigInt primitive.